webpack.config.js

```diff
@@ -270,7 +270,7 @@ module.exports = {
 		new DependencyExtractionWebpackPlugin( { injectPolyfill: true } ),
 	].filter( Boolean ),
 	watchOptions: {
-		ignored: '!packages/*/build-module/**/*',
+		ignored: /^(?!.*\/packages\/.*\/build-module\/.*)$/,
```

Comment:
I think this simply return `false` for every path except for an empty string?

```suggestion
		ignored: /^(?!.*\/packages\/.*\/build-module\/.*).*$/,
```

Maybe this is better? (That's enough regexp for today 😆)

Reply:
Great catch!

Before:
<img width="832" alt="Screen Shot 2021-03-30 at 13 02 43" src="https://user-images.githubusercontent.com/699132/112978982-483c6800-9158-11eb-84ab-fe132b2857de.png">

After:
<img width="843" alt="Screen Shot 2021-03-30 at 13 02 33" src="https://user-images.githubusercontent.com/699132/112979010-51c5d000-9158-11eb-8811-ac875c5c3943.png">